Loadsys/Sniffs/NamingConventions/ValidPrivateProtectedFunctionNameSniff.php

```diff
@@ -98,7 +98,7 @@ protected function processTokenWithinScope(PHP_CodeSniffer_File $phpcsFile, $sta
 		if ($isPublic === true) {
 			if ($methodName[0] === '_') {
 				$error = 'Public method name "%s" must not be prefixed with underscore';
-				$phpcsFile->addError($error, $stackPtr, 'PublicWithUnderscore', $errorData);
+				$phpcsFile->addWarning($error, $stackPtr, 'PublicWithUnderscore', $errorData);
 				return;
```

Comment:
I wasn't able to find any information to downgrade Errors into Warnings for the rulesset.xml file

Reading the docs: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml and https://github.com/squizlabs/PHP_CodeSniffer/wiki/Configuration-Options#changing-the-default-severity-levels

It looks like errors and warnings are only defined via the which they called as in the sniff itself, ie. `addError` vs. `addWarning`

Googling around also didn't provide any information.